tests/test_citations.py

```
@@ -0,0 +1,494 @@
+
+
+class TestPrintSummary:
+    """Tests for verification summary printing."""
+
+    def test_print_summary(self, capsys):
+        """Test summary output."""
+        # Re-import to ensure we have fresh references
+        from adversarial_workflow.utils.citations import (
+            URLResult,
+            URLStatus,
+            print_verification_summary,
```

Ruff (0.14.14)

[error] Redefinition of unused `print_verification_summary` from line 33: `print_verification_summary` redefined here

Remove definition: `print_verification_summary`

(F811)